modules/update-common-terms.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+
+= Common Terms
+
+Control Plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines. 
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service to view valid updates and update paths based on your current component versions and information in the graph.
+
+Machine Config Operator:: The _Machine Config Operator_ (MCO) is a cluster-level operator that manages the operating system and machine configurations. Through the MCO, platform administrators can configure and update systemd, CRI-O and Kubelet, the kernel, NetworkManager, and other system features on the nodes.
++
+For more information, see link:https://docs.openshift.com/container-platform/latest/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
+
+Recommended update edge:: _Recommended update edges_ are recommended updates between {product-title} releases.  Whether a given update is recommended may depend on the cluster's configured channel, current version, known bugs, and other information.
+
+Extended Update Support:: All post 4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS to EUS {product-title} releases that will cause fewer reboots of nodes.
++
+For more information, see link:https://access.redhat.com/support/policy/updates/openshift-eus[Red Hat OpenShift Extended Update Support (EUS) Overview].
+
+OpenShift Update Service:: The _OpenShift Update Service_ (OSUS) provides over-the-air updates to {product-title}, including {op-system-first}. It provides a graph, or diagram, that contains the vertices of component Operators and the edges that connect them.
```

Comment:
To clarify that OSUS doesn't deliver RHCOS in an update, but rather delivers an update _to_ RHCOS as well, maybe:
 
```suggestion
OpenShift Update Service:: The _OpenShift Update Service_ (OSUS) provides over-the-air updates to {product-title}, including to {op-system-first}. It provides a graph, or diagram, that contains the vertices of component Operators and the edges that connect them.
```